lib/utils/lifecycle.js

```diff
@@ -262,6 +262,28 @@ function runCmd_ (cmd, pkg, env, wd, stage, unsafe, uid, gid, cb_) {
     sh = process.env.comspec || 'cmd'
     shFlag = '/d /s /c'
     conf.windowsVerbatimArguments = true
+
+    // check if we have a composed command
+    // e.g.: `"test": "node ../test/test"`
+    if (cmd.indexOf(' ') !== -1) {
+      // split the command
+      var cmdChunks = cmd.split(' ')
+
+      cmd = cmdChunks.map(function (chunk) {
+        // let's assume that a path starts with either `./` or `../`
```

Comment:
Isn't this a wrong assumption? What if the path is absolute (E.g. `/my/folder/and/file.js` ?

Definitely using `path.parse()` (with `path.isAbsolute()`) would allow you to check if it's a valid path, and then `fs.statSync()` to check if it exists.